projects/canopy/src/lib/card/card.module.ts

```diff
@@ -11,13 +11,17 @@ import { LgCardHeaderComponent } from './card-header/card-header.component';
 import { LgCardSubtitleComponent } from './card-subtitle/card-subtitle.component';
 import { LgCardTitleComponent } from './card-title/card-title.component';
 import { LgCardComponent } from './card.component';
+import * as iconSet from '../icon/icons.interface';
```

Comment:
Maybe we should just import the chevron icon and not the whole set to help with tree shaking?

Reply:
I wondered if actually this should go into the consuming application and not this module. It ties the card module quite specifically to the use case of there being a back button in the header. There will be lots of scenarios where people won't need the chevron, but they'll be getting it loaded anyway.

Reply:
Took this out of the card.module as I also think this should go in the consuming app. The same is done with the Quick actions component icons and I also think we should keep consistency.